_data/speakers.yml

```diff
@@ -347,17 +360,18 @@
   image: profile_oam.jpg
   thumbnail: profile_oam.jpg
   alt: Osvaldo
-  topic:
-  proposal_summary:
+  topic: Sequential Monte Carlo: Introduction and diagnostics
+  proposal_summary: In this talk we will provide a brief introduction to Sequential Monte Carlo (SMC) methods and provide a guide to diagnose posterior samples computed using SMC.
   discourse_link:
   youtube_link:
   description: Osvaldo is a researcher at the National Scientific and Technical Research Council in Argentina
     and is notably the author of the book Bayesian Analysis with Python, whose second edition was published in December 2018.
     He also teaches bioinformatics, data science and Bayesian data analysis, and is a core developer of PyMC3 and ArviZ, and recently started contributing to Bambi.
     Originally a biologist and physicist, Osvaldo trained himself to python and Bayesian methods – and what he's doing with it is pretty amazing!
   presentation_type: Talk
-  time_zone:
+  time_zone: Americas
   track: Advanced
+  keywork: Osvtin
```

Comment:
Sequential monte carlo talk has 2 speakers, both should have the same keyword.

Reply:
Done